src/plugins/intel_npu/tests/functional/internal/overload/compiled_model/property.cpp

```diff
@@ -31,7 +31,8 @@ std::vector<std::pair<std::string, ov::Any>> exe_network_immutable_properties =
         {std::make_pair(ov::supported_properties.name(), ov::Any("deadbeef"))},
         {std::make_pair(ov::model_name.name(), ov::Any("deadbeef"))}};
 
-std::vector<std::pair<std::string, ov::Any>> properties = {{}};
+std::vector<std::pair<std::string, ov::Any>> driver_compiler_type_properties = {
+        {std::make_pair(ov::intel_npu::compiler_type.name(), ov::Any(ov::intel_npu::CompilerType::DRIVER))}};
```

Comment:
Do we have a strong reason to force the compiler_type to DRIVER here? DRIVER is default anyway and we want these tests to be executed with any compiler type selected by the plugin ( as default value or as a consequence of compatibility checks in the future).

Reply:
At first I didn't want to remove completely that vector of properties because that would break the tests. I believe that if it was left empty, the tests would fail.

But it seems like the parent class of these tests is used only in this file, so I will do the modifications to remove the third required property list.